about.html

```diff
@@ -0,0 +1,120 @@
+    </section>
+        <header class="h-desktop" id='menu'>
+            <i class="fa fa-times fa-2x" id='close-btn' aria-hidden="true"></i>
+            <ul class="nav-element">
+                <li><a class="nav-link" href="./index.html">Home</a></li>
+                <li><a class="nav-link" href="#">Program</a></li>
+                <li><a class="nav-link" href="#">News</a></li>
+            </ul>
+        
+        </header>
+    <section class="intro">
+        <div class="intro-content">
+            <h1>"VueJs Amsterdam"</h1>
+            <p class="intro-title">
+                WORLD'S MOST
+                SPECIAL AND
+                LARGEST VUE CONFERENCE
+            </p>
+            <section class="intro-desc">
+                Attend the Vue.js Community Event of the year and celebrate with Vue Communities
+            </section>
+            <span class="intro-contact">Please contact us per email for further questions about VueJs Amsterdam
+                <br><a href="">vue@amsterdam.com</a>
+            </span>
+        </div>
+    </section>
+    <section class="summit-logo">
+        <div class="program-title">
+            <h1>VueJs Amsterdam 2022</h1>
+            <span class="program-line"></span>
+        </div>
+        <div class="logo-desc">
+            <p>The logo of VueJs Amsterdam 2022 was decided through logo competition
+                from 8.June to 7.July</p>
+            <div class="logo">
+                <img src="./images/logo.png" alt="">
+            </div>
+            
+        </div>
+
+    </section>
+    <section class="past">
+        <div class="past-title">
+            <h1>See Past VueJs Amsterdam Summit</h1>
+            <span class="past-line"></span>
+        </div>
+        <h3>Take a look at the last two Web Summits</h3>
+        <div class="past-card">
+            <div class="past-container">
+                <h4 class="year"><b>2021</b></h4>
+                <p>VueJs 2021 Amsterdam</p>
+            </div>
+            <div class="past-container">
+                <h4 class="year">2018</h4>
+                <p>VueJs 2021 Amsterdam</p>
+            </div>
+        </div>
+    </section>
+        <section class="partner">
+            <div class="partner-title">
+                <h1>Partner</h1>
+                <span class="partner-line"></span>
+            </div>
+            <div class="partner-company">
+                <div class="company">
+                    <h5>GOOGLE</h5>
+                </div>
+                <div class="company">
+                    <h5>mozilla</h5>
+                </div>
+                <div class="company">
+                    <h5>NAVER</h5>
+                </div>
+                <div class="company">
+                    <h5>microverse</h5>
+                </div>
+            </div>
+        </section>
+        <footer class="footer">
+            VueJs Amsterdam 2022, &#169; Copyright Reserved.
+        </footer>
```

Comment:
- [ ] Hi this section should have a logo that is aligned to the left and text aligned to the right side of the footer.
![image](https://user-images.githubusercontent.com/18730106/152449792-c83f272b-b10e-480d-a54a-f8a393f4af03.png)
- [ ] For the about page the footer should have a dark background on the desktop version. please make this change